jlibra-core/src/main/java/dev/jlibra/admissioncontrol/query/Event.java

```diff
@@ -22,15 +23,27 @@
 
     long getSequenceNumber();
 
+    ByteSequence getMetadata();
+
     static Event fromGrpcObject(types.Events.Event event) {
         byte[] eventData = event.getEventData().toByteArray();
         try (DataInputStream eventDataStream = new DataInputStream(new ByteArrayInputStream(eventData))) {
             long amount = Deserialization.readLong(eventDataStream, 8);
             ByteSequence address = Deserialization.readByteSequence(eventDataStream, 32);
+            ByteSequence message = ByteSequence.from(new byte[0]);
+            if (eventDataStream.available() > 0) {
+                ByteSequence messageData = Deserialization.readByteSequence(eventDataStream, eventDataStream.available());
+                boolean onlyZerosStream = IntStream.range(0, messageData.toArray().length).parallel().allMatch(i -> messageData.toArray()[i] == 0);
```

Comment:
`parallel()` here looks like an overkill since there is no io nor heavy computation. Does it help anything?

Reply:
Yes, will remove it, doesn't make sense here.